### PR TITLE
First draft of Fungible token implementation

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces implementation of **fungible token** in caver-js.
For now, KIP7 is used but the name will be changed after KIP is created.

KIP7 inherites Contract.

Fungible token can be deployed by calling **caver.klay.KIP7.deploy**.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer https://github.com/klaytn/caver-js/issues/220
refer https://github.com/klaytn/caver-js/pull/227

## Further comments

The test category for KCT is created.

### CODE REVIEW

File: packages/caver-klay/caver-klay-kct/kctHelper.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            { indexed: true, name: 'owner', type: 'address' },
+            { indexed: true, name: 'spender', type: 'address' },
+            { indexed: false, name: 'value', type: 'uint256' },
+        ],
+        name: 'Approval',
+        type: 'event',
+    },
+]
+
+const kip7ByteCode =
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/kctHelper.js]: Please add a comment with the information below:
- source code location
- compilation options

File: packages/caver-klay/caver-klay-kct/kip7.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+*/
+
+const _ = require('lodash')
+
+const Contract = require('../caver-klay-contract')
+const { validateTokenInfoForDeploy, determineSendParams, kip7JsonInterface, kip7ByteCode } = require('./kctHelper')
+const { isAddress } = require('../../caver-utils')
+
+class KIP7 extends Contract {
+    static deploy(tokenInfo, deployer) {
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/kip7.js]: Please add a comment how to call this function with an example.

**Comment by jimni1222** [comment]
@kjhman21 
I'm sorry, but I needed a change that was reflected in dev branch, so I did force push after rebase.
I will comment on your comment if it is reflected.

**Comment by jimni1222** [comment]
For uint256 type parameters, the implementation must be modified to accept BigNumber.
Also, the value returned as the current string type should be converted to String / Number / BigNumber type according to the solidity return type.

For these additional implementations, i will close this PR now and generate it again after the implementation is complete.

**Pull request closed without merging**